### ISSUE TITLE
rebuild companion model after full respec

### ISSUE THREAD

**Issue by Gulli800** [BarleyFlour/WrathVisualAdjustmentsMod#83]
more of a request than an issue but didn't see requests except for pull request, don't know what that is so I put it here

in the beta (also in kingmaker) there was an option to rebuild companions model after full respec don't know why it disappeared in the full game mod,(my best guess would be model rebuild and grey skin rebuild got absorbed into one and something went wrong) if it would´t be too much trouble id love and apprised it being brought back when you are able to do so, I know and fully understand the grey skin problem take president over this

P.S, Thanks for the mod :)

**Comment by BarleyFlour** [comment]
You should be able to do this by clicking "Select Doll" & Then "Destroy Doll", no?

**Comment by Lord451125** [comment]
I think I may be having a similar issue.  Under Select Doll I'm not seeing a Destroy Doll option.  Only things I see that aren't sliders are the Use Normal past Portrait, and Preview under Custom Voice.

**Comment by Gulli800** [comment]
no, then the companion is invisible and only the gear can be seen

**Comment by Gulli800** [comment]
wait it appears after saving and loading the companion goes back to normal :)
thanks for the help and sorry I hope I wasn't a bother
(what should I do to the issue/request tread is there a solved button? sorry my first issue/request tread)

P.S. Thanks again for the mod :)
